C-Plus-Plus/README.md

```diff
@@ -63,6 +63,8 @@ Format: -[Program name](name of the file)
 
 -[Mathematical Series Demo](maths-series.cpp)
 
+-[Majority Element in Constant Space](https://practice.geeksforgeeks.org/problems/majority-element/0)
```

Comment:
Please do the required changes.  Everything else looks good.
```suggestion
-[Majority Element in Constant Space](Majority_Element.cpp)
```

Reply:
sure @niranjantuty